### ISSUE TITLE
Attempted import error: 'Annotation' is not exported from '@codemirror/state'.

### ISSUE THREAD

**Issue by athondapu** [codesandbox/sandpack#950]
# Bug report

## Packages affected
-   [ ] sandpack-react

## Description of the problem
I have an existing react project where it uses react version `"react": "^17.0.2"`. I am trying to use the `sandpack-react` in my react project but getting the following error while running the application.
```
./node_modules/@codesandbox/sandpack-react/dist/index.mjs
Attempted import error: 'Annotation' is not exported from '@codemirror/state'.
```
`sandpack-react` package version I am using is `"@codesandbox/sandpack-react": "^2.6.7"`

**Note:** The same package is working with the new project.

### What steps can we take to reproduce the problem?
1. Do an `npm install @codesandbox/sandpack-react` on the existing project which uses the older version of react.
2. Then just import the ```import {
    SandpackProvider,
    SandpackLayout,
    SandpackCodeEditor,
    SandpackFileExplorer,
    SandpackPreview,
  } from "@codesandbox/sandpack-react";``` this line.
3. Then run your app and it's start showing the above error.

### Your Environment

| Software                | Name/Version |
| ----------------------- | ------------ |
| Sandpack-client version |              |
| Sandpack-react version  |       ^2.6.7       |
| Browser                 |       Chromium       |
| Operating System        |      Ubuntu 22        |

**Comment by kishore0995** [comment]
Does your issue got resolved? I am getting same error for react v17.0

**Comment by danilowoz** [comment]
I'm closing this issue as it's been a while without any activity. Please feel free to report it again if the problem persists. Thanks so much!
